File: Plugins/Aspose_OCR_Java_for_Ruby/lib/asposeocrjava/OMR/setimageresolution.rb
module Asposeocrjava
  module SetImageResolution
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Initialize a string with template file location
        image_file = data_dir + "sample_omr.jpg"

        # Create an instance of OmrImage class and load the template using the factory method Load
        image = Rjb::import('com.aspose.omr.OmrImage').load(image_file)
        
        # Define new value of image resolution in double format
        image.setResolution(210.0) # overwrites the old DPI value

        # Do processing

        puts "Set image resolution."
    end
  end
end

File: Plugins/Aspose_OCR_Java_for_PHP/src/aspose/ocr/WorkingWithOMR/SetImageResolution.php
<?php
namespace Aspose\OCR\WorkingWithOMR;

use com\aspose\omr\OmrImage as OmrImage;
class SetImageResolution{

    public static function run($dataDir=null){

        # Initialize a string with template file location
        $image_file = $dataDir . "sample_omr.jpg";

        # Create an instance of OmrImage class and load the template using the factory method Load
        $omrImage=new OmrImage();
        $image = $omrImage->load($image_file);

        # Define new value of image resolution in double format
        $image->setResolution(210.0); # overwrites the old DPI value

        # Do processing

        print "Set image resolution.".PHP_EOL;

    }
}
?>